ipaserver/install/krbinstance.py

```diff
@@ -502,8 +502,11 @@ def issue_ipa_ca_signed_pkinit_certs(self):
             self._install_pkinit_ca_bundle()
             self.pkinit_enable()
         except RuntimeError as e:
-            logger.error("PKINIT certificate request failed: %s", e)
-            logger.error("Failed to configure PKINIT")
+            self.print_msg("Full PKINIT configuration was not succeed")
+            logger.debug(str(e))
```

Comment:
I think `e` will be converted to string anyway, will it not? Also, I think it makes sense to have the error message as a part of the warning.